src/main/resources/application.conf

```diff
@@ -175,6 +175,9 @@ mantis {
       net {
         peer-manager-timeout = 5.seconds
       }
+
+      # How many blocks back do we consider when searching for account recent transactions (eth_getAccountRecentTransactions)
+      tx-num-recent-blocks = 1000
```

Comment:
Maybe it's a good idea to allow users to send this value within the request instead. Maybe this config can be used as a default / max.

Reply:
makes sense